src/util/priceRate.ts

```diff
@@ -0,0 +1,10 @@
+const config = process.env
+
+export function getCurrencyPerSecond (): number {
```

Comment:
Lets use BigNumber for all of this calculation and return it since stream expects a BigNumber.